### PR TITLE
[script.skinshortcuts] 1.0.12

### PR DESCRIPTION
**Pull request by Ignoble61**
### Description
Update to the Skin Shortcuts script with various bug fixes and template improvements.

### Checklist:

- [ ] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [sciprt.foo.bar] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by Razzeee** [comment]
Any reason for not setting that first checkbox in the checklist?

**Comment by Ignoble61** [comment]
Because officially the script doesn't follow the official add-on rules as it writes an extra file to the skin's directory (script-skinshortcuts-includes.xml) in order for it to all, well, work. The script follows all other addon rules and the piracy policy, though.

Should have left a note about that in the op. My apologies.

**Comment by phil65** [comment]
update looks fine, thx.

File: script.skinshortcuts/resources/lib/gui.py [hunk 1]
```diff
@@ -807,6 +836,14 @@ def _save_properties( self, properties, labelIDChanges ):
                 # toSave[1] = property value
                 
                 saveData.append( [ self.group, property[0], toSave[0], toSave[1] ] )
+
+        # Add any default properties
+        for group in copyDefaults:
+            for defaultProperty in DATA.defaultProperties:
+                #[ groupname, itemLabelID, property, value ]
+                if defaultProperty[ 0 ] == group:
+                    log( repr( defaultProperty ) )
```
Comment (Ignoble61) [inline script.skinshortcuts/resources/lib/gui.py]: This line shouldn't have been included, and will be removed in the updated PR

**Comment by Ignoble61** [comment]
Note the PR has been updated, but the only change is removing a single logging line (originally resources/lib/gui.py line 845) which should have been removed before PR. There are no other changes to the code.

**Pull request merged**